cloning-revera.html

```diff
@@ -0,0 +1,305 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.9.0/css/all.css"
+    />
+    <style>
+    .test {
+      width: 40px;
+      height: 40px;
+      border: 3px solid white;
+    }
+
+   </style>
+</head>
+<body>
+
+
+    
+
+    
+    <nav class="nav justify-content-center navbar navbar-expand-lg navbar-light bg-light mb-3">
+        <p class="navbar-brand h1 mt-2 ">Revera</p>
+        <a class="nav-link active text-danger" aria-current="page" href="#">Home</a>
+        <a class="nav-link text-dark" href="#">About</a>
+        <a class="nav-link text-dark" href="#">Sample page</a>
+        <a class="nav-link text-dark" href="#">Portfolio</a>
+        <a class="nav-link text-dark" href="#">Blog</a>
+        <a class="nav-link text-dark" href="#">Parent page</a>
+        <a class="nav-link text-dark" href="#">Contact</a>
+      </nav>
+
+</header>
```

Comment:
The opening <header> is missing.
Also if you are going to use <header></header> it is a good idea to use <main></main> for the main content of the webpage